### PR TITLE
Feat/remove js sdk

### PR DESCRIPTION
**Pull request by LordCheta**
This PR aims to remove the JS SDK from this repository to it's own stand alone repo, found [here](https://github.com/rsksmart/rif-rollup-js-sdk)

### CODE REVIEW

File: core/bin/zksync_eth_sender/src/tests/mock.rs
Comment (antomor) [inline core/bin/zksync_eth_sender/src/tests/mock.rs:1]: @LordCheta why were those changes required for the JS SDK? I cannot see the relationship between them.

File: yarn.lock
Comment (antomor) [inline yarn.lock:1]: Do we need to update the yarn lock? I don't see changes in the dependencies in the package.json and recently we discovered an issue with an updated yarn.lock file, so if not required, I wouldn't change it here.

**Review by antomor — COMMENTED** [review]
@LordCheta Thank you for the PR. I've just left some questions about some changes that may not be required in this PR.

File: yarn.lock
Comment (LordCheta) [reply yarn.lock:1]: The lock file updated as I removed the sdk workspace and I also verified the project runs as expected, a couple packages were updated though. I do understand the need to be wary of such updates but thought it's best the team makes the decision of updating it or not.

File: core/bin/zksync_eth_sender/src/tests/mock.rs
Comment (LordCheta) [reply core/bin/zksync_eth_sender/src/tests/mock.rs:1]: I wondered about that too, this changes happened after I ran zk && zk init commands again after removing the sdk, upon confirming that there weren't any breaking changes( zk and zk init commands ran without errors, the server starts and I could still do the L1 -> L2 transactions and vice versa) I proceeded to include it in the PR so more knowledgeable rust engineers in the team can make a better decision.
